Build/apf v6 build.sh

IgorMinar opened the pull request.

IgorMinar committed:
feat: update the package output of build.sh to APF v6

mary-poppins commented:
You can preview d4b68f6 at https://pr22808-d4b68f6.ngbuilds.io/.

mary-poppins commented:
You can preview 06c1a17 at https://pr22808-06c1a17.ngbuilds.io/.

mary-poppins commented:
You can preview a50cdfe at https://pr22808-a50cdfe.ngbuilds.io/.

mary-poppins commented:
You can preview 6c2283c at https://pr22808-6c2283c.ngbuilds.io/.

mary-poppins commented:
You can preview 3ff3cef at https://pr22808-3ff3cef.ngbuilds.io/.

mary-poppins commented:
You can preview a559c82 at https://pr22808-a559c82.ngbuilds.io/.

jasonaden reviewed (approved):
Looks good. Travis failure.

mary-poppins commented:
You can preview 37942f1 at https://pr22808-37942f1.ngbuilds.io/.

mary-poppins commented:
You can preview 383bf7e at https://pr22808-383bf7e.ngbuilds.io/.

mary-poppins commented:
You can preview ad53a01 at https://pr22808-ad53a01.ngbuilds.io/.

IgorMinar commented:
The PR still fails for two reasons:
- size regression likely due to some optimization problems when processing the unflattened esm5 input
- e2e failures from aio examples - this seems to be related to the update of CLI from head (a0fc67002f9578d72de20e626ebdf839ab966571)

mary-poppins commented:
You can preview 1749954 at https://pr22808-1749954.ngbuilds.io/.

mary-poppins commented:
You can preview e71928f at https://pr22808-e71928f.ngbuilds.io/.

IgorMinar commented on packages/common/http/testing/package.json:
```diff
@@ -1,7 +1,8 @@
+  "module": "../../fesm5/http/testing.js",
+  "es2015": "../../fesm2015/http/testing.js",
+  "esm5": "../../esm5/http/testing.js",
+  "esm2015": "../../esm2015/http/testing.js"
```
missing fesm5 and fesm2015

IgorMinar committed:
test: add workaround to cli-hello-world payload-size test

See https://github.com/angular/devkit/pull/524

IgorMinar committed:
test: improve ng build flags for cli-hello-world

Remove unnecessary --build-optimizer flag and add --sourcemaps flag to
generate source maps for better debugging.

IgorMinar committed:
build: don't mark language-service package as sideEffect free

we are not sure if this is the case and we don't need this package to
be optimized at the moment.

mary-poppins commented:
You can preview a1c2d13 at https://pr22808-a1c2d13.ngbuilds.io/.

mary-poppins commented:
You can preview d0d5f26 at https://pr22808-d0d5f26.ngbuilds.io/.

mary-poppins commented:
You can preview 0204d71 at https://pr22808-0204d71.ngbuilds.io/.

mary-poppins commented:
You can preview 75df8a8 at https://pr22808-75df8a8.ngbuilds.io/.

mary-poppins commented:
You can preview 0f10c24 at https://pr22808-0f10c24.ngbuilds.io/.

mary-poppins commented:
You can preview b688ef8 at https://pr22808-b688ef8.ngbuilds.io/.
